src/js/utils/strings.ts

```diff
@@ -94,25 +95,29 @@ export function seconds(str, frameRate) {
 }
 
 // Convert an offset string to a number; supports conversion of percentage offsets
-export function offsetToSeconds(offset, duration, frameRate) {
+export function offsetToSeconds(offset: string | number, duration?: number, frameRate?: number): number | null {
     if (isString(offset) && offset.slice(-1) === '%') {
         const percent = parseFloat(offset);
         if (!duration || !isValidNumber(duration) || !isValidNumber(percent)) {
             return null;
         }
         return duration * percent / 100;
     }
-    return seconds(offset, frameRate);
+
+    if (frameRate) {
```

Comment:
what is the type error we get without this conditional compared to the above `seconds(offset, frameRate)`

Reply:
Nice catch! There isn't one. I forgot to remove this when I removed the method overloads.